lib/Net/SAML2/AttributeConsumingService.pm

```diff
@@ -66,8 +66,8 @@ sub to_xml {
             index     => $self->index,
             isDefault => $self->default,
         },
-        $xml->ServiceName($self->namespace, undef, $self->service_name),
-        $self->_has_service_description ? $xml->ServiceDescription($self->namespace, undef, $self->service_description) : (),
+        $xml->ServiceName($self->namespace, { 'xml:lang' => 'en' }, $self->service_name),
```

Comment:
Perhaps instead of hardcoding it to 'en' we can add it as an attribute (and default it to 'en'), so others can define their own language, eg, 'it', 'nl' etc? 

@timlegge  wdyt?

I could also accept this and change it in another commit.

Reply:
Yeah I think we do need a way to specify a language but as @MartijnVdS pointed out it is hard coded in the generate_metadata so lets approve her and add an issue to fix everywhere.